playbooks/roles/mysql/files/edxapp.sql

```diff
@@ -101,21 +101,22 @@ VALUES
   1,
   1
 );
+commit;
 
 /* Insert an entry so that new site is enabled with a domain name */
 
-INSERT INTO django_site (domain,name) VALUES ('courses.microsoft.com','courses');
+/* INSERT INTO django_site (domain,name) VALUES ('courses.microsoft.com','courses'); 
```

Comment:
Rather than commenting out, should this just be removed?

Reply:
yes, i just removed it.